packages/react-charting/src/utilities/HTMLTooltipText.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { hasOverflow, ITooltipHostProps, TooltipHost, TooltipOverflowMode } from '@fluentui/react';
+import { getAccessibleDataObject } from './index';
+import { IAccessibilityProps } from '../types/index';
+import { Async } from '../Utilities';
+
+interface IHTMLTooltipTextProps {
+  className?: ITooltipHostProps['hostClassName'];
+  content?: ITooltipHostProps['content'];
+  accessibilityData?: IAccessibilityProps;
+}
+
+interface IHTMLTooltipTextState {
+  textOverflow: boolean;
+}
+
+export class HTMLTooltipText extends React.Component<IHTMLTooltipTextProps, IHTMLTooltipTextState> {
+  private _tooltipChild = React.createRef<HTMLSpanElement>();
+  private _resizeObserver?: ResizeObserver;
+  private _async: Async;
+
+  constructor(props: IHTMLTooltipTextProps) {
+    super(props);
+
+    this.state = {
+      textOverflow: false,
+    };
+
+    this._async = new Async(this);
+  }
+
+  public render(): React.ReactNode {
+    const { className, content, accessibilityData } = this.props;
+
+    return (
+      <TooltipHost overflowMode={TooltipOverflowMode.Self} hostClassName={className} content={content}>
+        <span
+          {...getAccessibleDataObject(accessibilityData)}
+          ref={this._tooltipChild}
+          data-is-focusable={this.state.textOverflow}
+          tabIndex={this.state.textOverflow ? 0 : -1}
```

Comment:
Add a comment that tabIndex is to fix the focusing delay while navigating elements.

Reply:
removed the attribute